apps/holidays-viewer/templates/holidays.html

```diff
@@ -0,0 +1,76 @@
+<html><body><div class="mw-parser-output">
+<div class="card mx-auto mb-1"><div class="card-body p-2"><b>test holiday</b></div></div>
+<!-- 
+NewPP limit report
```

Comment:
What is this limit report here? Where is this coming from and do we need it?

Reply:
> What is this limit report here? Where is this coming from and do we need it?

This is the Template and Lua limit report. This report does not have any usages regarding to this app. We can remove it.